### DISCUSSION TITLE
Introducing Keploy OSS Fund

### DISCUSSION THREAD

**Discussion by Swpn0neel** [keploy/keploy#2352] [Show and tell]
At Keploy, we are starting a fund to support open-source projects and maintainers. We're giving ~$12,500 per developer per year, in alignment with and inspired by Sentry's [OSS Pledge](https://osspledge.com/) initiative.

## Why are we starting an Open-Source Fund?

Today, Keploy has over 5.2K Stars, 850+ Contributors, 900 Forks, and thousands of contributions made, all with the help, support, and love of the open-source community. And, with the Keploy OSS Fund, our goal is to help establish a precedent for giving early and often — to reinforce that sponsorship should be the norm, rather than the exception.

- **Giving back to the community:** This financial power meant many things, such as Appwrite could build a team, invest in a better product offering, community swag, and start an OSS fund. And all of it with one thing in mind, giving back to the community. Helping those who have helped us.

- **Hiring from the OSS community:** The first thing Keploy did with the funding was build out the team. And what better place to find the right people than in the open-source community? We hired some of the first and top Keploy contributors. Sarthak Shyngle, Charan Kamarapu and Gourav Kumar. They formed Keploy’s founding engineering team.

## How are we funding?

We will then ensure that candidate projects meet two additional criteria:

- **Availability on GitHub Sponsors:** We're a small team of engineers looking to minimize administrative overhead. We've reached out to a few proposed projects that aren't available via GitHub Sponsors, but this excludes some candidates that we'd like to support in the future.

- **Lack of conflicts:** We avoided projects that are maintained by Astral team members.

The goal of this fund is to thank, encourage, and empower maintainers. Some maintainers offer additional features or support in exchange for funding — we think that's a good model, but we want to keep this fund separate from our business needs, so we'll make exchanges for goods and services in addition to this fund.

## Who are we funding?

We're giving between $20 and $150 per month per project/maintainer. With these amounts, we're balancing the number of projects we can sponsor with an amount that is meaningful to maintainers.

## How can you apply?

Comment down about your project and details in a below format :  

```
Name:  Animesh Pathak
GitHub: Sonichigo
Project: Keploy
Description: Keploy is an Open Source Testing ToolKit for developers
Amount: 25$
Usage: To cover the cost of xyz.
```

**Comment by shivamsouravjha** [comment]
Name: Shivam Sourav Jha
Github: Shivamsouravjha
Project: StockSight
Description: StockSight helps investors make smarter decisions by analyzing mutual fund portfolios to reveal underlying stock investments. By streamlining data access through factsheet uploads or email parsing, it reduces friction for users. The platform also aggregates key financial metrics like profit & loss and peer comparisons to offer reliable insights, addressing the challenges of user drop-off and inconsistent online data.
Amount: 150$
Usage: to get TAC Verification
